src/core/Toast/Toast.tsx

```diff
@@ -150,32 +149,64 @@ export const Toast = (props: ToastProps) => {
           }}
         >
           <div ref={onRef}>
-            <div className={`iui-toast iui-${category}`}>
-              <div className='iui-status-area'>
-                {<StatusIcon className='iui-icon' />}
-              </div>
-              <div className='iui-message'>{content}</div>
-              {link && (
-                <a className='iui-anchor' onClick={link.onClick}>
-                  {link.title}
-                </a>
-              )}
-              {(type === 'persisting' || hasCloseButton) && (
-                <IconButton
-                  size='small'
-                  styleType='borderless'
-                  onClick={close}
-                  aria-label='Close'
-                >
-                  <SvgCloseSmall />
-                </IconButton>
-              )}
-            </div>
+            {ToastPresentation({
```

Comment:
Looks weird...Why not `<ToastPresentation />`?

Reply:
Oh, all I did was a VSCode refactor. Fixed now